tests/neg/i13960.check

```diff
@@ -0,0 +1,6 @@
+-- [E108] Declaration Error: tests/neg/i13960.scala:13:10 --------------------------------------------------------------
+13 |    case A() => // error
+   |         ^^^
+   |          [31mA[0m is not a valid result type of an unapply method of an [35mextractor[0m.
```

Comment:
I don't think our checkfiles have ansi color escape codes normally

Reply:
I didn't investigate why we're getting it here, in part because it auto-generated and passes...